content/environment_variables.tex

```diff
@@ -30,14 +30,34 @@
     \tabularnewline\hline
 %%
 \EnvVarDecl{SHMEM\_SYMMETRIC\_SIZE}
-    & Non-negative integer
-    & Number of bytes to allocate for symmetric heap
+    & Non-negative decimal value (integer or floating point) with an optional
+    character suffix
```

Comment:
Does "decimal" mean base-10 or that it has a decimal point?  The former seems unnecessary and the latter contradicts the parenthetical "integer" comment.  Can we shorten this to "Non-negative integer or floating point value with an optional character suffix"?

Reply:
fixed